core/bridges/orm.go

```diff
@@ -64,12 +64,13 @@ func (o *orm) DeleteBridgeType(bt *BridgeType) error {
 // BridgeTypes returns bridge types ordered by name filtered limited by the
 // passed params.
 func (o *orm) BridgeTypes(offset int, limit int) (bridges []BridgeType, count int, err error) {
-	if err = postgres.NewQ(o.db).Get(&count, "SELECT COUNT(*) FROM bridge_types"); err != nil {
+	q := postgres.NewQ(o.db, postgres.WithLogger(o.logger))
```

Comment:
This ideally would be refactored to use a readonly transaction wrapping those two queries